db/seeds.rb

```diff
@@ -5,3 +5,60 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+require 'open-uri'
+require 'nokogiri'
+puts "🔫 Destroy everything ..."
+# User.destroy_all
+# Products.destroy_all
+
+puts "🧑 Seed hardcoded users ..."
+barcode_sample = [4901330574352, 4954835290708, 4582469493006, 4902508070546, 4902777088785, 4902106010067, 4976548100075]
+# https://www.jancode.xyz/4901330574352/
+# https://www.jancode.xyz/4954835290708/
+
+
+barcode_sample = [4901330574352]
+barcode_sample.each do |barcode|
+  url = "https://www.jancode.xyz/#{barcode}/"
+  html_file = URI.open(url).read
+  html_doc = Nokogiri::HTML(html_file)
+  links = []
+  product_info = {}
+  test = []
```

Comment:
When I read code I need to look through the whole process to understand what is "test" is for. Would be nice if the variable name itself telling its own purpose.